tools/cgcollector2/src/CallGraphCollectionAction.cpp

```diff
@@ -0,0 +1,182 @@
+/**
+ * File: CallGraphCollectionAction.cpp
+ * License: Part of the MetaCG project. Licensed under BSD 3 clause license. See LICENSE.txt file at
+ * https://github.com/tudasc/metacg/LICENSE.txt
+ */
+
+#include <fstream>
+#include <iostream>
```

Comment:
Why do we need `<iostream>` here? Can't we rely on LLVM implementation?
Also prefer to put STL includes at the end.